docs/reference/edot-sdks/android/manual-instrumentation.md

```diff
@@ -0,0 +1,90 @@
+---
+navigation_title: Manual instrumentation
+description: Manually instrument your Android application using the Elastic Distribution of OpenTelemetry Android.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/android/current/manual-instrumentation.html
+---
+
+# Manual instrumentation
+
+You can create your custom spans, metrics, and logs, using the [OpenTelemetry SDK APIs](https://opentelemetry.io/docs/languages/java/api/#opentelemetry-api), which you can find in the [OpenTelemetry](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/OpenTelemetry.html) object provided through the `getOpenTelemetry()` method. 
+
+Alternatively, for common operations, you can use the [convenience agent extensions](#convenience-extensions) to create telemetry in a less verbose way.
+
+## OpenTelemetry APIs
+
+After completing the [setup](getting-started.md) process, the SDK has configured an [OpenTelemetry](https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-api/latest/io/opentelemetry/api/OpenTelemetry.html) object for you, which is available through the `getOpenTelemetry()` method. 
```

Comment:
I've noticed that we call our EDOT different names across the docs (EDOT Android, SDK, Elastic Distribution of OpenTelemetry Android, and I tend to call it `agent` in the existing docs, because it's short and to avoid overloading the word "SDK", as it's also used when talking about the OpenTelemetry Java SDK). I'm not particularly fixed on a specific way to call it, but I think we should be consistent across the docs. So I'd suggest picking one name and using it everywhere.